Sources/StreamChatUI/ChatMessageList/Reactions/ChatMessageDefaultReactionsBubbleView.swift

```diff
@@ -5,6 +5,9 @@
 import StreamChat
 import UIKit
 
+public typealias DefaultChatReactionPickerBubbleView = ChatMessageDefaultReactionsBubbleView
+
+/// This component will be renamed to `DefaultChatReactionPickerBubbleView` in future major versions.
 open class ChatMessageDefaultReactionsBubbleView: ChatReactionPickerBubbleView {
```

Comment:
```suggestion
@available(*, deprecated, renamed: "DefaultChatReactionPickerBubbleView") // would be also nice to move it to Deprecations.swift
public typealias ChatMessageDefaultReactionsBubbleView = DefaultChatReactionPickerBubbleView

open class DefaultChatReactionPickerBubbleView: ChatReactionPickerBubbleView {
```

Reply:
We can't actually do this, the deprecated message will surface to the typelalias as well, which is stupid 😓